src/aktualizr_lite/version.h

```diff
@@ -0,0 +1,10 @@
+#include <string>
+
+#include <string.h>
+
+struct Version {
+  std::string raw_ver;
+  Version(const std::string& version) : raw_ver(std::move(version)) {}
```

Comment:
from clang-tidy:

```
aktualizr/src/aktualizr_lite/version.h:7:49: error: std::move of the const variable 'version' has no effect; remove std::move() or make the variable non-const [performance-move-const-arg,-warnings-as-errors]
  Version(const std::string& version) : raw_ver(std::move(version)) {}
```

Reply:
Sorry about always getting clang-tidy stuff wrong. I can't seem to find a way to run clang-tidy that produces the same results as TravisCI. I've tried installing clang-6 into to docker container and I think I've also tried clang-7. From a "clean" code base both versions produces lots of warnings. I need to just sit down and figure out how I can get this to work in the standard docker container.

Reply:
No worries, this version problem is indeed annoying. If you run it from `Dockerfile.debian.testing`, you should be quite close from what's going on on CI.

e.g: `./scripts/run_docker_test.sh docker/Dockerfile.debian.testing`, then `ninja clang-tidy` from a CMake directory or directly `./scripts/test.sh` with the flags used in the CI job.